database/schema.sql

```diff
@@ -167,6 +167,35 @@ CREATE TABLE updating_email_lists_additions (
   PRIMARY KEY (listname, email)
 );
 
+-- ROTATION TABLES
+
+CREATE TABLE rotation_prefrosh (
+  prefrosh_id INTEGER NOT NULL AUTO_INCREMENT,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL,
+  preferred_name VARCHAR(255),
+  image LONGBLOB,
+  bucket ENUM('-2', '-1', '000', '0', '0.5', '1', '1.5', '2', '3') NOT NULL DEFAULT '0',
+  votes_neg_two INTEGER NOT NULL DEFAULT 0,
+  votes_neg_one INTEGER NOT NULL DEFAULT 0,
+  votes_zero INTEGER NOT NULL DEFAULT 0,
+  votes_plus_one INTEGER NOT NULL DEFAULT 0,
+  votes_plus_two INTEGER NOT NULL DEFAULT 0,
+  votes_plus_three INTEGER NOT NULL DEFAULT 0,
+  dinner INTEGER,
+  attended_dinner BOOLEAN,
+  comments VARCHAR(5000),
```

Comment:
Might as well make this VARCHAR(65535), since the string length will need to be 2 bytes anyway.